sips/sip-138.md

```diff
@@ -0,0 +1,48 @@
+---
+sip: 138
+title: Update Volume Tracking Event
+status: Proposed
+author: David Schwartz (@dvd-schwrtz)
+discussions-to: Discord
+created: 2021-05-17
+---
+
+## Simple Summary
+
+The goal of this proposal is to allow for tracking of volume sources on L2 and to improve tracking of volume sources on L1
+
+## Abstract
+
+This SIP proposes the `ExchangeTracking` event add a fee param for tracking volume sources on Synthetix protocol
+
+## Motivation
+
+At the moment we can only retrieve the volume sources on L1 and not L2. Also, the tracking of data on L1 is more complicated than it needs to be.
+
+## Specification
+
+### Overview
+
+On L1 we use fee reclamation, which comes with a waiting period. When an exchange happens, an `ExchangeTracking` event is fired, followed at the end of the waiting period by an `ExchangeEntryAppended` event to finalize the fee reclamation process. In order to track volume sources, we need data from both of these events; however, since there is no fee reclamation on L2 we are unable to track volume sources.
+
+This SIP proposes to modify the `ExchangeTracking` event to add a fee param which will remove the need for the `ExchangeEntryAppended` event to be used in volume source calculations on L1 or L2.
+
+### Rationale
+
+We cannot capture volume source data right now without a fee reclamation period unless we emit extra data that is already available in the `ExchangeTracking` event.
+
+### Technical Specification
+
+https://github.com/Synthetixio/synthetix/pull/1139/files -> The PR to update the `ExchangeTracking` event has already been written. It is ready to merge pending approval of this SIP. The only external site that might be affected by this change is the cross asset swaps by Curve; we will reach out to discuss with them before merging. Internally, we will make sure stats volume source section remains accurate when we merge this PR as well.
```

Comment:
Can you just add the diff of the events that you are changing here? Events are part of interfaces and it's worth showing the diff here rather than just linking to the PR.